doc/architecture.rst

```diff
@@ -1094,6 +1094,8 @@ See `Erasure Code Notes`_ for additional details.
 Cache Tiering
 -------------
 
+.. note:: Cache tiering is deprecated in Reef.
```

Comment:
Almost wonder if s/is/was/ for futureproofing.

Reply:
I think that we can assume the historical present for the sake of posterity.